docs/pages/includes/node-logins.mdx

```diff
@@ -0,0 +1,30 @@
+When Teleport's Auth Service receives a request to list Teleport Nodes (e.g., to
+display Nodes in the Web UI or via `tsh ls`), it only returns the Nodes that the
+current user is authorized to access.
+
+For each Node in the user's Teleport cluster, the Auth Service applies the
+following checks in order and, if one check fails, hides the Node from the user:
+
+{/*
+
+ TODO: We might want to mention that the Auth Service checks the resource's
+ namespace as well, but we currently do not document resource namespaces.
+ This would not be the appropriate place to include our only namespace
+ documentation. See gravitational/teleport issue #12580.
+ 
+*/}
+
+- None of the user's roles contains a `deny` rule that matches the Node's labels.
+- None of the user's roles contains a `deny` rule that matches the user's
```

Comment:
I may be misunderstanding what this is referring to, but I'm not sure this applies to listing nodes.

Reply:
I was attempting to paraphrase [`RoleSet.CheckAccess`](https://github.com/gravitational/teleport/blob/bda45f78b0c742c2bbfad60f7cc757a99c99a556/lib/services/role.go#L1854-L1856), which looks like it would check if one of the `matchers`, which I thought would be a `services.loginMatcher` in the case of a Node, [matches a `deny` rule](https://github.com/gravitational/teleport/blob/bda45f78b0c742c2bbfad60f7cc757a99c99a556/lib/services/role.go#L1903-L1913).